ydb/core/statistics/ut/ya.make

```diff
@@ -25,7 +25,7 @@ SRCS(
     ut_aggregator.cpp
     ut_statistics.cpp
     ut_save_load_stats.cpp
-    ut_statistics_service.cpp
+#    ut_statistics_service.cpp
```

Comment:
Can you explain what is the reason to disable this test?

Reply:
currently there is a strange problem with one of the tests in this suite. we discussed it and decided to postpone the investigation